apps/demos/Demos/Chat/FileAttachments/description.md

```diff
@@ -0,0 +1,23 @@
+DevExtreme Chat supports file attachments. When this feature is activated, an “Attach” button appears in the message input field, allowing users to add files to their messages. 
+
+To enable file attachments in Chat, specify [fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile**.
```

Comment:
Please update so that this is technically correct. I tried to shorten the intro.

```suggestion
DevExtreme Chat supports file attachments. When this feature is activated ([fileUploaderOptions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#fileUploaderOptions).**uploadFile** is `true`), an “Attach” button appears in the message input field, allowing users to add files to their messages. 
```